proxy/server.go

```diff
@@ -24,6 +53,10 @@ type proxySrv struct {
 	target da.DA
 }
 
+func (p *proxySrv) Config(ctx context.Context, request *pbda.ConfigRequest) (*pbda.ConfigResponse, error) {
+	return &pbda.ConfigResponse{MaxBlobSize: DefaultMaxBytes}, nil
```

Comment:
just call `return da.DA.Config()`. proxy's Config is its implementation's Config.

Reply:
Done